collections/_posts/2023-07-06-Mean_teachers_are_better_role_models.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: review
+title: Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results
```

Comment:
The title should be within quotes so that it can properly interpreted by jekyll:
```suggestion
title: "Mean teachers are better role models: Weight-averaged consistency targets improve semi-supervised deep learning results"
```